testing-plugin/src/main/kotlin/arrow/meta/plugin/testing/TestingLibraryWrapper.kt

```diff
@@ -10,11 +10,11 @@ import java.nio.file.Paths
 
 private const val DEFAULT_FILENAME = "Example.kt"
 
-internal data class CompilationResult(
+data class CompilationResult(
```

Comment:
Instead of doing CompilationResult public, I'd think about the desired assertion. The purpose of this library is to provide a simple testing framework. If we publish the compilationResult, it seems this library doesn't make sense and we might use the kotlin compiler testing library directly.

Reply:
I am not writing an assertion

Reply:
I need the `testing-library` with the wrappers we added in `testing-library`. For the `IDE` it's a utility.

Reply:
Please, could you give more details?

Reply:
About what specifically?

Reply:
About the use of the library in IDE...